app/jobs/document_proofing_job.rb

```diff
@@ -12,8 +12,7 @@ def perform(
     trace_id:,
     image_metadata:,
     analytics_data:,
-    flow_path:,
-    liveness_checking_enabled: nil
```

Comment:
Since this is a background job we will need to keep this argument here. Otherwise when old hosts spawn a job we'll get an argument error when a new worker picks it up. We'll need to leave this in here until all the deployed hosts are running code that does not use this argument.